tensorflow/lite/micro/tools/make/Makefile

```diff
@@ -261,10 +272,10 @@ PRJDIR := $(GENDIR)prj/
 MICRO_LITE_EXAMPLE_TESTS := $(shell find $(TENSORFLOW_ROOT)tensorflow/lite/micro/examples/ -maxdepth 2 -name Makefile.inc)
 
 # Internal examples are copied outside the TFLM repo in google.
-ifeq ($(TENSORFLOW_ROOT),)
-  MICRO_LITE_EXAMPLE_TESTS += $(shell find ../google/ -name Makefile_internal.inc)
+ifneq ($(EXTERNAL_DIR),)
+  MICRO_LITE_EXAMPLE_TESTS += $(shell find $(EXTERNAL_DIR) -name Makefile_internal.inc)
 else
-  MICRO_LITE_EXAMPLE_TESTS += $(shell find google/ -name Makefile_internal.inc)
+  MICRO_LITE_EXAMPLE_TESTS += $(shell find ../google -name Makefile_internal.inc)
```

Comment:
let's add a comment:
```
TODO(b/): remove this once the internal examples are modified to make use of EXTERNAL_DIR instead of replying on google being hard-coded.
```

Reply:
Done. b/246833632